### ISSUE TITLE
Bug in Version 1.4: Placement of scripts does not work

### ISSUE THREAD

**Issue by wotis** [nodecode/wp-minify-fix#5]
Version 1.4 now puts some scripts in the footer, too. This is the new recommended default-setting. But this does not work for all scripts. After upgrading I found my site broken (e.g. my lightbox-script does not work when placed in the footer).

I wanted to change the option to "place all scripts in header", but here seems to be a bug, because this setting has no effect. Some scripts are still in the footer and the site is still broken.

Had to disable js-compression completely for the moment to make my site work again.

Please fix this and make the placement-option work. Thank you!

**Comment by nodecode** [comment]
Mysterious, for me, everything seems to work perfectly. But I'll look at that in more detail.

**Comment by wotis** [comment]
Thank you. If you need any further informations or something I can look up for you in my installation please let me know.

**Comment by wotis** [comment]
I did some research and I think I found the reason for the problems. I noticed, that with the new version 1.4 all jquery based scripts did not work.

I saw, that I had the string "jquery" in the field "JavaScript files to exclude from minify". I removed it from there - and most of the scripts worked again after activating js-compression.

The placement-setting was "In the header, and if necessary also in the footer" and the lightbox/foobox still did not work. I now changed it to "Everything in the header" which made even the lightbox work again.

so everthing seems to work now. it seems not to be a bug in the plugin.
